sendgrid/helpers/mail/validators.py

```diff
@@ -0,0 +1,68 @@
+################################################################
+# Various types of Validators
+################################################################
+
+class ValidateAPIKey(object):
+    """Validates content to ensure SendGrid API key is not present"""
+
+    regexes = None
+
+    def __init__(self, regex_strings=list(), use_default=True):
```

Comment:
Mutable data structures are usually frowned upon as default arguments.

* http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments

Could this be rewritten as shown in the link below (using `None` as the default arg)?

* http://docs.python-guide.org/en/latest/writing/gotchas/#what-you-should-do-instead

Reply:
Fixed it! Interesting idea and something I'll implement in my future project/commits - I had no idea about that but it makes sense